### DISCUSSION TITLE
基于move合约的开源图书系统Dapp

### DISCUSSION THREAD

**Discussion by Labbicorn** [NonceGeek/Move-Camp#188] [Amazing Things!]
今天突然发现zlibrary 不能登陆好像被被美国ban，在想是不是可以在区块链上 实现这样的一个app。文件存储使用arweare。
业务逻辑使用move合约实现。

**Comment by Demian101** [comment]
挺好的想法，我可以帮忙写前端

**Comment by Labbicorn** [comment]
哇塞 好呀好呀可以一起实现



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Demian ***@***.***&gt;
发送时间: 2022年11月4日 17:45
收件人: NonceGeek/Web3-dApp-Camp ***@***.***&gt;
抄送: hacker8964 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [NonceGeek/Web3-dApp-Camp] 基于move合约的开源图书系统Dapp (Discussion #188)





 
挺好的想法，我可以帮忙写前端
